Dockerfile

```diff
@@ -6,7 +6,7 @@ RUN apt-get update && \
     apt-get autoremove -y && \
     apt-get clean
 
-RUN npm install -g pm2
+RUN npm install -g pm2 babel-cli babel-preset-es2015
```

Comment:
Minor thing: but in general it's better to keep the presets in the devDependencies of your package, such that they will just be installed with `npm install` for anyone who wants to contribute to the project.